internal/session/session.go

```diff
@@ -288,30 +290,41 @@ func (s *Session) startReader(ctx context.Context, readerExit chan any) {
 		return
 	}
 
-	lastPackageTimestamp := &time.Time{}
-	*lastPackageTimestamp = time.Now()
+	s.processReaderResponse(resp, goroutineCloseCh, pipeFinishCh)
+}
+
+func (s *Session) processReaderResponse(resp *http.Response, goroutineCloseCh, pipeFinishCh chan any) {
+	lastPackageTimestamp := time.Now()
```

Comment:
there is a race condition with `lastPackageTimestamp`..

Reply:
@eahydra @cardyok Fixed